package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Show GBFS stations to a GeoJSON feature collection.",
   "main": "index.js",
   "scripts": {
+    "download": "babel-node download.js --presets es2015,stage-2",
```

Comment:
We use `mastarm` on our projects to avoid needing to install the same babel libraries on each project. Also, in this case I wrote `download.js` in a way that specifically doesn't need babel if you're running node v6+. I'd rather add the node requirement of 6+ and not babel as a dependency.